rskj-core/src/main/java/org/ethereum/rpc/exception/RskJsonRpcRequestException.java

```diff
@@ -57,4 +57,12 @@ public static RskJsonRpcRequestException blockNotFound(String message) {
     public static RskJsonRpcRequestException stateNotFound(String message) {
         return new RskJsonRpcRequestException(-32600, message);
     }
+
+    public static RskJsonRpcRequestException unknownBLockException(){
+        return new RskJsonRpcRequestException(-39001, "Unknown block");
+    }
+
+    public static RskJsonRpcRequestException headerNotFound(){
```

Comment:
just to be consistent with `eth_call`, can we reuse `RskJsonRpcRequestException::blockNotFound()`?

Reply:
I just aligned the error codes with Geth, maybe we should check eth_call and verify it.

Reply:
well, I believe the problem wasn't with message itself but rather with the fact that internal server error was being returned. I guess it's fine if we make use / re-use the `eth_call`'s message

Reply:
they have different error codes, it is ok with that?

Reply:
I guess so, error code standardisation is out of scope, plus error codes themselves seem to be out of spec and client specific